com.unity.render-pipelines.core/Editor/MaterialUpgrader.cs

```diff
@@ -57,6 +65,36 @@ class KeywordFloatRename
         }
         List<KeywordFloatRename> m_KeywordFloatRename = new List<KeywordFloatRename>();
 
+        /// <summary>
+        /// Type of property to rename.
+        /// </summary>
+        public enum RenameType
```

Comment:
```suggestion
        public enum MaterialPropertyType
```
?

Reply:
Done in 3fa71b7caa76812686402193099bf796e231609f.